src/test/java/com/circlek/Test01.java

```diff
@@ -10,33 +10,41 @@
 
 public class Test01 {
 
-    //CODE REVIEW
+    //CODE REVIEW_Ilya
     @Test
-    public void assertTextOnPage(){
+    public void assertTextOnPage() {
         System.setProperty("webdriver.chrome.driver", "lib/chromedriver.exe");
         WebDriver driver = new ChromeDriver();
         driver.get("https://circlek-public.github.io/cki-service-recruitment-task/");
 
         //assert Remember Me Button
+
         WebElement rememeberMeLabel = driver.findElement(By.id("remember_me_label"));
+
         assertEquals("Remember Me", rememeberMeLabel.getText());
 
         //assert Sign In text
+
         WebElement signInButton = driver.findElement(By.className("login"));
+
         assertEquals("Sign In", signInButton.getText());
     }
 
     @Test
-    public void checkRememberMeCheckBox(){
+    public void checkRememberMeCheckBox() {
         System.setProperty("webdriver.chrome.driver", "lib/chromedriver.exe");
         WebDriver driver = new ChromeDriver();
         driver.get("https://circlek-public.github.io/cki-service-recruitment-task/");
 
         //click Remember Me checkbox
+
         WebElement rememberMeCheckBox = driver.findElement(By.xpath("//*[contains(text(), \"Remember Me\")]/../input"));
+
         rememberMeCheckBox.click();
```

Comment:
The best way is to replace it into particular method in PageObject class and make waitUntilElementIsClickable before so that we can be sure that we wait enough time before element became clickable
```suggestion
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(timeout));
        wait.until(ExpectedConditions.elementToBeClickable(element));
        rememberMeCheckBox.click();
```
but obviously our WebDriverWait actions should being a global method in a separate class